tests/moe/test_trtllm_gen_fused_moe.py

```
@@ -982,6 +983,155 @@ def get_tolerances(self):
+        }
+
+    def quantize_inputs(self, hidden_states, *unused_args):
+        """No scaling for hidden states."""
+        return {
+            "hidden_states": hidden_states.to(torch.bfloat16),
+            "hidden_states_scale": None,
+        }
+
+    def prepare_static_weights_for_kernel(
+        self,
+        args_dequant,
```

Ruff (0.14.3)

Unused method argument: `args_dequant`

(ARG002)